dashboard/static/css/layout.css

```diff
@@ -7,9 +7,31 @@ body {
     margin: 0 0 100px;
 }
 
+main {
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+a {
+    color: #00549E;
+}
+
+form {
+    margin: 20px 0;
+}
+
+footer {
+    height: 100px;
+    width: 100%;
+}
```

Comment:
I'm not sure if we need this. The footer looks the same for me without it, at least.